### PR TITLE
adds a kms key with minimum policies to logging acccount

### PR DESCRIPTION
**Pull request by masafari**

### CODE REVIEW

File: UPGRADING.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# Upgrading to 0.16.x
```
Comment (marwinbaumannsbp) [inline UPGRADING.md:1]: Since its not a breaking change lets tag this 0.15.1, can you move this text to changelog.md ?

File: logging.tf [hunk 2]
```diff
@@ -82,3 +82,80 @@ resource "aws_s3_account_public_access_block" "logging" {
...
+  source              = "github.com/schubergphilis/terraform-aws-mcaf-kms?ref=v0.2.0"
+  providers           = { aws = aws.logging }
+  name                = "logging"
+  description         = "KMS key to use with logging account"
+  enable_key_rotation = true
+  policy              = data.aws_iam_policy_document.kms_key_logging.json
+  tags                = var.tags
+}
+
+data "aws_iam_policy_document" "kms_key_logging" {
```
Comment (marwinbaumannsbp) [inline logging.tf:96]: i'll give you a call about this

Comment (marwinbaumannsbp) [reply logging.tf:96]: discussed and OK

File: UPGRADING.md (hunk 1 above)
Comment (masafari) [reply UPGRADING.md]: updated!

**Review by marwinbaumannsbp — APPROVED** [review]

**Pull request merged**